developer/src/kmcmplib/src/kmcmpdll.h

```diff
@@ -3,17 +3,17 @@
 #include <compfile.h>
 
 namespace kmcmp {
-  KMX_BOOL AddCompileString(LPSTR buf);
+  KMX_BOOL AddCompileWarning(LPSTR buf);
   extern BOOL FWarnDeprecatedCode;
   extern int currentLine;
 }
 
-KMX_BOOL AddCompileMessage(KMX_DWORD msg);
+KMX_BOOL AddCompileError (KMX_DWORD msg);
 
 // TODO: These macros can return FALSE in functions that expect a DWORD CERR_x
 //       return value type. This is just plain wrong!
-#define SetError(err)       { if(AddCompileMessage(err) || (err & CERR_FATAL)) return FALSE; }
-#define AddWarning(warn)    { if(AddCompileMessage(warn)) return FALSE; }
+#define SetError(err)       { if(AddCompileError (err) || (err & CERR_FATAL)) return FALSE; }
+#define AddWarning(warn)    { if(AddCompileError (warn)) return FALSE; }
```

Comment:
Should this be this?

```suggestion
#define AddWarning(warn)    { if(AddCompileWarning (warn)) return FALSE; }
```

Reply:
does not work with AddCompileWarning